scripts/shared/content-import.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+STARTERKIT_VERSION="v1.0.0"
```

Comment:
Instead of specifying this version, let's just point this to the latest release programatically. If we want, we can make this an optional cmd line argument if someone wants to specify a version.